src/key_value_store/client/kvstore_client.h

```diff
@@ -0,0 +1,35 @@
+#include "../KeyValueStoreInterface.h"
+
+#include <grpcpp/grpcpp.h>
+#include "store.grpc.pb.h"
+
+#include <string>
+#include <vector>
+
+#include "../core/KeyValueStore.h"
+
+using grpc::Channel;
+using grpc::ClientContext;
+using grpc::ClientReaderWriter;
+using grpc::Status;
+
+using kvstore::PutRequest;
+using kvstore::PutReply;
+using kvstore::GetRequest;
+using kvstore::GetReply;
+using kvstore::RemoveRequest;
+using kvstore::RemoveReply;
+
+class KeyValueStoreClient final : public KeyValueStoreInterface {
```

Comment:
Add a comment for the class, all member methods, and member vars.  (Here and elsewhere.)

Reply:
done